docs/changelog.txt

```diff
@@ -39,6 +39,7 @@ changelog.txt uses a syntax similar to RST, with a few special sequences:
 - Fix crash for some players when they launch DF outside of the Steam client
 
 ## Misc Improvements
+- `Autonick`: additional nicknames based on burrowing animals, colours, gems and minerals added
```

Comment:
`autonick` should be in lowercase (like the command), otherwise the docs builder won't find the link